overlord/snapstate/snapstate.go

```diff
@@ -57,6 +57,12 @@ const (
 	// 0x40000000 >> iota
 )
 
+const (
+	// WithSideInfo makes the code look for .sideinfo files for sideloaded
+	// snaps (used in first-boot)
+	WithSideInfo = 0x40000000 >> iota
```

Comment:
given that this is for SnaSetup only we probably want a comment  before const as you had in the rollback branch and also we could type it with SnapSetupFlags